tests/testthat/_snaps/apply_fmt_fn.md

```diff
@@ -12,19 +12,19 @@
       apply_fmt_fn(dplyr::mutate(ard_fmt_checks, fmt_fn = list("xoxo", "xoxo")))
     Condition
       Error in `dplyr::mutate()`:
-      i In argument: `stat_fmt = map2(...)`.
+      i In argument: `stat_fmt = pmap(...)`.
       Caused by error:
-      ! The format "xoxo" is not valid.
+      ! The format "xoxo" for `fmt_fn` is not valid for the value 20.090625 for the statistic "mean". String must begin with 'x' and only consist of x's, a single period or none, and may end with a percent symbol.
```

Comment:
@ddsjoberg is something like this and the example right below what you had in mind?